cmd/performance-profile-creator/cmd/root.go

```diff
@@ -18,27 +18,197 @@ package cmd
 
 import (
 	"fmt"
+	"io/ioutil"
 	"os"
+	"path"
+	"path/filepath"
 
 	"github.com/spf13/cobra"
 	yaml "gopkg.in/yaml.v2"
+	k8syaml "k8s.io/apimachinery/pkg/util/yaml"
+	"k8s.io/component-helpers/scheduling/corev1"
 
 	"k8s.io/utils/pointer"
 
 	performancev2 "github.com/openshift-kni/performance-addon-operators/api/v2"
+	machineconfigv1 "github.com/openshift/machine-config-operator/pkg/apis/machineconfiguration.openshift.io/v1"
+	v1 "k8s.io/api/core/v1"
+
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
+const (
+	// ClusterScopedResources defines the cluster scope resources under
+	// must-gather-dir/quay-io-openshift-kni-performance-addon-operator-must-gather*
+	ClusterScopedResources = "cluster-scoped-resources"
+	// CoreNodes defines substring in the path to obtain node data
+	CoreNodes = "core/nodes"
+	// MCPools defines substring in the path to obtain mcp data
+	MCPools = "machineconfiguration.openshift.io/machineconfigpools"
+	// YAMLSuffix defines yaml suffix
+	YAMLSuffix = ".yaml"
```

Comment:
can we possibly get also .yml files there ?  (minus the `a`)

Reply:
@fromanirh As per Yanir's question, Does must gather create YAMLs with .yml extension as well? Is it okay to assume that the extension is yaml only for now?

Reply:
well I took that the other way around, are we expecting extensions different from `.yaml` at all?

Reply:
yep